documentation/projects/proposals/trust_and_safety/content_report_moderation/20230411-project_proposal_content_report_moderation.md

```diff
@@ -0,0 +1,338 @@
+# 2023-04-11 Project Proposal
+
+## Reviewers
+
+<!-- Choose two people at your discretion who make sense to review this based on their existing expertise. Check in to make sure folks aren't currently reviewing more than one other proposal or RFC. -->
+
+- [ ] @zackkrida
+- [ ] @olgabulat
+
+## Project summary
+
+<!-- A brief one or two sentence summary of the project's features -->
+
+While Openverse supports reporting results as mature, we do not have a process
+of handling these reports. Likewise, there are significant gaps in our
+infrastructure that make it difficult to respond to these reports or to lean on
+contributors outside the core maintainers for help processing them. This project
+will address these gaps, make it possible to easily and safely respond to
+content reports, and develop a baseline process for moderating content based on
+reports. It will also rename the "mature content" reports to "sensitive content"
+reports. This project is not the end-all, be-all for content report moderation
+in Openverse. The assumption should be that future iterations will occur. This
+project will establish the baseline practices that we will iterate on. It will
+not address or even mention every current or future improvement possible or
+required. It is necessary to constrain the scope in this way, otherwise this
+project would simply have no ending. In the same vein as "improve search" will
+always be an aspect of our ongoing projects, so will "improve content safety".
+
+In addition to the previous broad qualification, this project explicitly does
+not cover the following with the understanding that future projects will:
+
+- Sensitive content disputes: allowing users to dispute if they disagree with
+  moderation decisions, especially allowing creators to dispute their works
+  marked as sensitive.
+- A process for reviewing DMCA reports. There is significant overlap in the
+  changes to the reports queue, but this project will not address how to handle
+  these reports, only sensitive content ones.
+
+## Goals
+
+<!-- Which yearly goal does this project advance? -->
+
+Content safety and moderation.
+
+## Requirements
+
+<!-- Detailed descriptions of the features required for the project. Include user stories if you feel they'd be helpful, but focus on describing a specification for how the feature would work with an eye towards edge cases. -->
+
+Requirements are outlined here, with further details below. Each of these
+roughly correspond to a single implementation plan. Requested plans are
+[covered in detail below](#required-implementation-plans).
+
+1. Language is updated to use "sensitive" or "sensitive content" rather than
+   "mature" where possible.
+1. Volunteers outside the core maintainers can access content reports in Django
+   admin and work through a queue of pending reports.
+1. Bulk actions can be taken to deindex or mark sensitive many results.
+1. Use a computer vision API to analyse images on a report to give hints to
```

Comment:
I would suggest to allow the user to control if they want to see the images blurred or not.
Making all images blurred by default and without the option to turn this off can make the moderation work slower.